.changeset/hot-dodos-flow.md

```diff
@@ -0,0 +1,5 @@
+---
+'openzeppelin-solidity': minor
+---
+
+Replace safety transfer acceptance check calls with `Address.functionCall`
```

Comment:
I'm not sure if we'll require a changeset because this is not changing anything for the users or developers. CC: @frangio

Reply:
The guidelines say:

>All changes to the core codebase (excluding tests, auxiliary scripts, etc.) must be documented in a changelog, except for purely cosmetic or documentation changes.

This is not cosmetic or documentation. Refactors should also be described in the changelog. The reasoning  is that users may see that a contract was updated in X.Y.Z and may want to see exactly what changed.